.github/workflows/ci.yaml

```
@@ -0,0 +1,26 @@
+name: test
```

```suggestion
name: CI
```

Makes it the same as dynamiclistener (and norman with the opened PR), that label appears in the github UI, would be great to be consistent imo.